packages/app/cypress/e2e/specs_list_latest_runs.cy.ts

```diff
@@ -0,0 +1,570 @@
+import defaultMessages from '@packages/frontend-shared/src/locales/en-US.json'
```

Comment:
@mike-plummer this seems to be at a good stage for you to review when you get the chance